admin-dev/themes/new-theme/scss/theme.scss

```diff
@@ -73,3 +73,8 @@ $color-icons: #6c868e;
   float:none;
   margin:0 auto;
 }
+
+.fullspace {
```

Comment:
This is an helper to force the docs page loaded in an HTML5 <object> to take all the available space.